### ISSUE TITLE
 librosa.core.load is significantly slower than scipy.io.wavfile.read

### ISSUE THREAD

**Issue by EgorLakomkin** [SeanNaren/deepspeech.pytorch#40]
I have observed that loading wav files through  librosa.core.load is slower than with scipy's wavfile.read function.

IHere are two examples(same setup: LibriSpeech train, batch size 64, two GPU 1080, single layer GRU 1024 hidden units, 4 workers ):

with librosa.core.load
```
Epoch: [1][800/4395]	Time 0.534 (0.571)	Data 0.002 (0.215)	Loss 177.7152 (132.5715)	
Epoch: [1][801/4395]	Time 2.908 (0.574)	Data 2.372 (0.218)	Loss 178.4938 (132.6288)	
Epoch: [1][802/4395]	Time 1.918 (0.576)	Data 1.382 (0.219)	Loss 170.5456 (132.6761)	
Epoch: [1][803/4395]	Time 1.359 (0.577)	Data 0.821 (0.220)	Loss 172.4971 (132.7257)	
Epoch: [1][804/4395]	Time 1.007 (0.577)	Data 0.473 (0.221)	Loss 173.2764 (132.7761)	
Epoch: [1][805/4395]	Time 2.924 (0.580)	Data 2.387 (0.223)	Loss 170.5575 (132.8231)	
Epoch: [1][806/4395]	Time 1.580 (0.581)	Data 0.993 (0.224)	Loss 171.3481 (132.8709)	
Epoch: [1][807/4395]	Time 1.711 (0.583)	Data 1.173 (0.225)	Loss 171.6878 (132.9190)	
Epoch: [1][808/4395]	Time 1.234 (0.584)	Data 0.684 (0.226)	Loss 180.7255 (132.9781)	
Epoch: [1][809/4395]	Time 1.770 (0.585)	Data 1.232 (0.227)	Loss 165.3421 (133.0181)	
Epoch: [1][810/4395]	Time 3.278 (0.588)	Data 2.740 (0.230)	Loss 166.3688 (133.0593)	
Epoch: [1][811/4395]	Time 1.039 (0.589)	Data 0.503 (0.231)	Loss 174.8942 (133.1109)	
Epoch: [1][812/4395]	Time 0.936 (0.589)	Data 0.398 (0.231)	Loss 173.7952 (133.1610)	
Epoch: [1][813/4395]	Time 2.226 (0.591)	Data 1.685 (0.233)	Loss 177.2127 (133.2152)	
Epoch: [1][814/4395]	Time 3.571 (0.595)	Data 3.034 (0.236)	Loss 167.6428 (133.2575)	
Epoch: [1][815/4395]	Time 0.548 (0.595)	Data 0.002 (0.236)	Loss 169.2704 (133.3017)	
Epoch: [1][816/4395]	Time 0.867 (0.595)	Data 0.330 (0.236)	Loss 166.2436 (133.3420)	
Epoch: [1][817/4395]	Time 2.086 (0.597)	Data 1.538 (0.237)	Loss 181.1502 (133.4006)	
Epoch: [1][818/4395]	Time 3.903 (0.601)	Data 3.295 (0.241)	Loss 171.9818 (133.4477)	
Epoch: [1][819/4395]	Time 0.543 (0.601)	Data 0.002 (0.241)	Loss 175.6589 (133.4993)	
Epoch: [1][820/4395]	Time 0.545 (0.601)	Data 0.002 (0.241)	Loss 174.9106 (133.5498)	
Epoch: [1][821/4395]	Time 2.316 (0.603)	Data 1.776 (0.242)	Loss 168.7184 (133.5926)	
Epoch: [1][822/4395]	Time 3.403 (0.607)	Data 2.852 (0.246)	Loss 174.2117 (133.6420)	
Epoch: [1][823/4395]	Time 0.700 (0.607)	Data 0.164 (0.246)	Loss 166.9638 (133.6825)	
Epoch: [1][824/4395]	Time 0.548 (0.607)	Data 0.002 (0.245)	Loss 183.0482 (133.7424)	
Epoch: [1][825/4395]	Time 2.556 (0.609)	Data 1.995 (0.247)	Loss 170.0922 (133.7865)
```
and with scipy.io.wavfile.read

```
Epoch: [1][800/4395]	Time 0.998 (0.486)	Data 0.002 (0.005)	Loss 182.8188 (137.1815)	
Epoch: [1][801/4395]	Time 0.988 (0.486)	Data 0.002 (0.005)	Loss 182.8905 (137.2385)	
Epoch: [1][802/4395]	Time 0.978 (0.487)	Data 0.002 (0.005)	Loss 177.0392 (137.2881)	
Epoch: [1][803/4395]	Time 0.966 (0.488)	Data 0.002 (0.005)	Loss 176.3289 (137.3368)	
Epoch: [1][804/4395]	Time 0.993 (0.488)	Data 0.002 (0.005)	Loss 177.1688 (137.3863)	
Epoch: [1][805/4395]	Time 0.951 (0.489)	Data 0.002 (0.005)	Loss 173.7921 (137.4315)	
Epoch: [1][806/4395]	Time 0.999 (0.490)	Data 0.002 (0.005)	Loss 176.1973 (137.4796)	
Epoch: [1][807/4395]	Time 0.995 (0.490)	Data 0.002 (0.005)	Loss 176.3397 (137.5278)	
Epoch: [1][808/4395]	Time 0.968 (0.491)	Data 0.002 (0.005)	Loss 184.6344 (137.5861)	
Epoch: [1][809/4395]	Time 0.997 (0.491)	Data 0.002 (0.005)	Loss 170.4413 (137.6267)	
Epoch: [1][810/4395]	Time 0.951 (0.492)	Data 0.002 (0.005)	Loss 169.0714 (137.6655)	
Epoch: [1][811/4395]	Time 1.005 (0.493)	Data 0.002 (0.005)	Loss 176.8274 (137.7138)	
Epoch: [1][812/4395]	Time 0.998 (0.493)	Data 0.002 (0.005)	Loss 178.3829 (137.7639)	
Epoch: [1][813/4395]	Time 1.010 (0.494)	Data 0.002 (0.005)	Loss 181.8083 (137.8181)	
Epoch: [1][814/4395]	Time 1.006 (0.494)	Data 0.002 (0.005)	Loss 168.5279 (137.8558)	
Epoch: [1][815/4395]	Time 0.990 (0.495)	Data 0.002 (0.005)	Loss 171.7180 (137.8973)	
Epoch: [1][816/4395]	Time 1.025 (0.496)	Data 0.002 (0.005)	Loss 169.9290 (137.9366)	
Epoch: [1][817/4395]	Time 1.006 (0.496)	Data 0.002 (0.005)	Loss 183.6840 (137.9926)	
Epoch: [1][818/4395]	Time 0.992 (0.497)	Data 0.002 (0.005)	Loss 176.9357 (138.0402)	
Epoch: [1][819/4395]	Time 1.038 (0.498)	Data 0.002 (0.005)	Loss 180.7055 (138.0923)	
Epoch: [1][820/4395]	Time 0.994 (0.498)	Data 0.002 (0.005)	Loss 179.9813 (138.1434)	
Epoch: [1][821/4395]	Time 1.000 (0.499)	Data 0.002 (0.005)	Loss 171.4475 (138.1839)	
Epoch: [1][822/4395]	Time 1.025 (0.499)	Data 0.002 (0.005)	Loss 180.0951 (138.2349)	
Epoch: [1][823/4395]	Time 0.989 (0.500)	Data 0.002 (0.005)	Loss 172.6573 (138.2767)	
Epoch: [1][824/4395]	Time 1.010 (0.501)	Data 0.002 (0.005)	Loss 185.8300 (138.3345)	
Epoch: [1][825/4395]	Time 1.040 (0.501)	Data 0.002 (0.005)	Loss 175.1313 (138.3791)	

```

**Comment by SeanNaren** [comment]
Wow awesome find man! My assumption is this is a straight forward call replace, am I correct in thinking this?

**Comment by EgorLakomkin** [comment]
Though on a smaller batch_size or a larger number of the workers this could be avoided.

**Comment by ryanleary** [comment]
Can be closed based on #48.
